addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14448,13 +14458,13 @@ msgstr ""
 #. Description of settings category "Videos -> File lists" with label #14081
 #: system/settings/settings.xml
 msgctxt "#36176"
-msgid "Category containing settings for how video file lists are handled."
+msgid "This category contains the settings for how video file lists are handled."
 msgstr ""
 
 #. Description of setting "Videos -> File lists -> Default select action" with label #22079
 #: system/settings/settings.xml
 msgctxt "#36177"
-msgid "Toggle between Choose, Play (default), Resume and Show Information. Choose will select an item, e.g. open a directory in files mode. Resume will automatically resume videos from the last position that you were viewing them, even after restarting the system."
+msgid "Toggle between \"Choose\", \"Play\" (default), \"Resume\" and \"Show information\". \"Choose\" will select an item (e.g. open a directory in files mode.) \"Resume\" will automatically resume videos from the last position that you were viewing them, even after restarting the system."
```

Comment:
I'm not opposed to them, but didn't check how it'll look like in action. It also depends on the skin and Confluence is handling this very badly (lines are way too long). Maybe we should not use line breaks in descriptions for now and first come up with a concept on how to improve this stuff in future (we could also make the option values bold f.e.). So probably better no line breaks this time but at least fix Confluence and make the text box less wide (and maybe add a small info icon next to it to fill the empty space).